code/index.html

```diff
@@ -1,33 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Grandstander:wght@300;500;600&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Bangers&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Creepster&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Princess+Sofia&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap" rel="stylesheet">
+  <title>Movie Bot</title>
+</head>
 
-  <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
-    <main>
-      <section class="chat" id="chat"></section>
+<body>
+  <h1>Welcome to Movie Bot</h1>
+  <h2>Get inspiration on which movie to watch</h2>
+  <main id="main">
+    <section class="chat" id="chat"></section>
 
-      <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <input id="name-input" type="text" />
-          <button class="send-btn" type="submit">
-            Send
-          </button>
-        </form>
+    <div class="input-wrapper" id="input-wrapper">
+      <form id="name-form">
+        <input id="name-input" type="text" />
+        <button class="send-btn" type="submit">
+          Send
+        </button>
+      </form>
+      <div class="genre-options" id="genre-options">
+        <button id="comedy-btn" class="comedy-btn">Comedy</button>
+        <button id="action-btn" class="action-btn">Action</button>
+        <button id="horror-btn" class="horror-btn">Horror</button>
+        <button id="romantic-btn" class="romantic-btn">Romantic</button>
       </div>
```

Comment:
A better way to archive this functionality is to use `innerHTML`. This approach is not secure because your user can easily remove the hide class with one mouse click in dev tools.